tests/Http/Event/ImportEventRequestHandlerTest.php

```diff
@@ -41,8 +41,10 @@ protected function setUp(): void
             $this->documentImporter,
             $this->uuidGenerator,
             new CallableIriGenerator(fn (string $eventId) => 'https://io.uitdatabank.dev/events/' . $eventId),
-            new CallableIriGenerator(fn (string $eventId) => 'https://io.uitdatabank.dev/places/' . $eventId),
             new CombinedRequestBodyParser(
+                new LegacyEventRequestBodyParser(
+                    new CallableIriGenerator(fn (string $placeId) => 'https://io.uitdatabank.dev/places/' . $placeId)
```

Comment:
Just a small nit (and I just noticed this now), but maybe we should use the production URLs here. I know it doesn't really matter but might be a bit cleaner since production is the most important environment and then it is at least "correct" there, just in case it might matter at some point in the future?

Reply:
Ah nevermind this is a test 🙈

Reply:
Yeah, I have the habit now of using `io.uitdatabank.dev` for tests instead of `example.com` or something like that.